gradoop-examples/src/main/java/org/gradoop/benchmark/cypher/CypherBenchmark.java

```diff
@@ -0,0 +1,263 @@
+  /**
+   * Option do declare path to statistics
+   */
+  private static final String OPTION_STATISTICS_PATH = "s";
+  /**
+   * Option to declare path to statistics csv file
+   */
+  private static final String OPTION_CSV_PATH = "c";
+  /**
+   * Option to declare verification
+   */
+  private static final String OPTION_QUERY = "q";
+  /**
+   * Option for used first name in query.
+   */
+  private static final String OPTION_FIRST_NAME = "n";
+  /**
+   * Used input path
+   */
+  private static String INPUT_PATH;
+  /**
+   * Used output path
+   */
+  private static String OUTPUT_PATH;
+  /**
+   * Used to indicate if statistics are used
+   */
+  private static boolean HAS_STATISTICS;
+  /**
+   * Used for statistics path
+   */
+  private static String STATISTICS_INPUT_PATH;
+  /**
+   * Used csv path
+   */
+  private static String CSV_PATH;
+  /**
+   * Used query
+   */
+  private static String QUERY;
+  /**
+   * Used first name for query (q1,q2,q3)
+   */
+  private static String FIRST_NAME;
+
+  static {
+    OPTIONS.addOption(OPTION_INPUT_PATH, "input", true,
+      "Path to source files.");
+    OPTIONS.addOption(OPTION_OUTPUT_PATH, "output", true,
+      "Path to output file");
+    OPTIONS.addOption(OPTION_CSV_PATH, "csv", true,
+      "Path to csv statistics");
+    OPTIONS.addOption(OPTION_QUERY, "query", true,
+      "Used query (q1,q2,q3,q4,q5,q6)");
+    OPTIONS.addOption(OPTION_FIRST_NAME, "query-name", true,
+      "Used first Name in Cypher Query");
+    OPTIONS.addOption(OPTION_STATISTICS_PATH, "statistics", true,
+      "Path to previously generated statistics.");
+  }
+
+  /**
+   * Main program to run the benchmark. Arguments are the available options.
+   *
+   * @param args program arguments
+   * @throws Exception IO or execution Exception
+   */
+  public static void main(String[] args) throws Exception {
+    CommandLine cmd = parseArguments(args, SubgraphBenchmark.class.getName());
+
+    if (cmd == null) {
+      System.exit(1);
+    }
+
+    // test if minimum arguments are set
+    performSanityCheck(cmd);
+
+    // read cmd arguments
+    readCMDArguments(cmd);
+
+    ExecutionEnvironment env = getExecutionEnvironment();
+    GradoopFlinkConfig config = GradoopFlinkConfig.createConfig(env);
+
+    // read graph
+    DataSource source = new IndexedCSVDataSource(INPUT_PATH, config);
+    LogicalGraph graph = source.getLogicalGraph();
+
+    // prepare collection
+    GraphCollection collection;
+
+    // get cypher query
+    String query = getQuery(QUERY);
+
+    // execute cypher with or without statistics
+    if (HAS_STATISTICS) {
+      GraphStatistics statistics = GraphStatisticsHDFSReader
+        .read(STATISTICS_INPUT_PATH, new Configuration());
+
+      collection = graph.query(query, statistics);
+    } else {
+      collection = graph.query(query);
+    }
+
+    // write data to sink
+    DataSink sink = new JSONDataSink(OUTPUT_PATH, config);
```

Comment:
`JsonDataSink` is deprecated.

Reply:
Take a look at https://github.com/dbs-leipzig/gradoop/blob/68efc3d7f2037e02d8e0e0062096bcb9421ffdba/gradoop-examples/src/main/java/org/gradoop/examples/AbstractRunner.java#L145-L149
The format used there will be changed to csv in #905.